src/main/kotlin/com/google/androidstudiopoet/BuildGradle.kt

```diff
@@ -38,9 +38,6 @@ object BuildGradle {
             "sourceCompatibility = \"1.8\"\n" +
             "targetCompatibility = \"1.8\"\n" +
             "buildscript {\n" +
-            (if (useKotlin)
-                "    ext.kotlin_version = '1.1.51'\n"
-            else "")+
```

Comment:
removed it because it's already defined in the top build.gradle
no need to update it to a wrong version